scripts/a8sidecar.sh

```diff
@@ -63,3 +63,10 @@ tar -xzf /tmp/a8sidecar-${A8SIDECAR_RELEASE}-linux-amd64.tar.gz -C /
 #Cleanup
 rm -rf ${A8TMP}
 rm /tmp/a8sidecar-${A8SIDECAR_RELEASE}-linux-amd64.tar.gz
+
+if [ $HAVE_WGET -eq 1 ]; then
```

Comment:
I suggest we not including dumb-init in our installation script, as it is _not_ a dependency for the amalgam8 sidecar.
The sidecar can perfectly operate without it, because it properly reaps its child processes, and so does nginx.

However, given that using a proper init process is considered best-practices when running multiple processes in a container, we should install and use it our prebuilt sidecar images, and probably add a doc recommendation so that users who build their own images will install dumb-init/runit/whatever, and refer them to our Dockerfile as an example.